velox/docs/functions/spark/string.rst

```diff
@@ -178,6 +178,19 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
         SELECT startswith('js SQL', 'SQL'); -- false
         SELECT startswith('js SQL', null); -- NULL
 
+.. spark:function:: str_to_map(string, entryDelim, keyValueDelim) -> map(string, string)
+
+    Returns a map by splitting ``string`` into entries with ``entryDelim`` and splitting
+    each entry into key/value with ``keyValueDelim``.
+    Only supports constant single-character ``entryDelim`` and ``keyValueDelim``. Throws
```

Comment:
Is any single-character allowed to be entry and key-value delimeter in Spark? Do we have limitations for their content?

Reply:
Not found the requirement in spark document. Assume there is no special requirement.